packages/collector/Collector_Db.php

```diff
@@ -0,0 +1,90 @@
+<?php
+function collector_dump_db($zip)
+{
+	$tables   = collector_get_db_tables();
+	$mysqli   = collector_get_db();
+	$sqlFile  = collector_get_tmpfile('schema', 'sql');
+	$tmpFiles = [$sqlFile];
+
+	file_put_contents($sqlFile, sprintf("-- %s\n", json_encode(['SECTION START' => 'SCHEMA'])), FILE_APPEND);
+
+	foreach($tables as $table)
+	{
+		file_put_contents($sqlFile, sprintf("-- %s\n", json_encode(['ACTION' => 'DROP', 'TABLE' => $table])), FILE_APPEND);
+		file_put_contents($sqlFile, sprintf("DROP TABLE IF EXISTS `%s`;\n", $table), FILE_APPEND);
+		file_put_contents($sqlFile, sprintf("-- %s\n", json_encode(['ACTION' => 'CREATE', 'TABLE' => $table])), FILE_APPEND);
+		file_put_contents($sqlFile, preg_replace("/\s+/", " ", collector_dump_db_schema($table)) . "\n", FILE_APPEND);
+	}
+
+	file_put_contents($sqlFile, sprintf("-- %s\n", json_encode(['SECTION END' => 'SCHEMA'])), FILE_APPEND);
+	file_put_contents($sqlFile, sprintf("-- %s\n", json_encode(['SECTION START' => 'RECORDS'])), FILE_APPEND);
+
+	// Process in reverse order so wp_users comes before wp_options
+	// meaning the fakepass will be cleared before transients are
+	// dumped to the schema backup in the zip
+	foreach(array_reverse($tables) as $table)
+	{
+		file_put_contents($sqlFile, sprintf("-- %s\n", json_encode(['ACTION' => 'INSERT', 'TABLE' => $table])), FILE_APPEND);
+		$recordList = collector_dump_db_records($table);
+
+		while($record = $recordList->fetch_assoc())
+		{
+			if($table === 'wp_users' && (int) $record['ID'] === (int) wp_get_current_user()->ID)
+			{
+				$record['user_pass'] = wp_hash_password(collector_use_fakepass());
+			}
+
+			$insert = sprintf(
+				'INSERT INTO `%s` (%s) VALUES (%s);',
+				mysqli_real_escape_string($mysqli, $table),
+				implode(', ', array_map(fn($f) => "`" . mysqli_real_escape_string($mysqli, $f) . "`", array_keys($record))),
```

Comment:
Let's use `$wpdb` instead of `mysqli_*` functions whenever possible. Ideally there won't be a single `mysqli_*` call at all. WordPress may use different MySQL PHP extensions so there's no guarantee this function will be available.